ukov4/uko_v4.go

```diff
@@ -177,7 +179,7 @@ func (uko *UkoV4) ListManagedKeysWithContext(ctx context.Context, listManagedKey
 	builder := core.NewRequestBuilder(core.GET)
 	builder = builder.WithContext(ctx)
 	builder.EnableGzipCompression = uko.GetEnableGzipCompression()
-	_, err = builder.ResolveRequestURL(uko.Service.Options.URL, `/api/v4/managed_keys`, nil)
+	_, err = builder.ResolveRequestURL(uko.Service.Options.URL, `/v4/managed_keys`, nil)
```

Comment:
The lack of /api was causing problems in the previous version, and I believe I pushed a change to the API spec about that. It will probably need to be in the new version too.